src/elements/scholarship-block.html

```diff
@@ -0,0 +1,222 @@
+            <div class="number-text">Diversity and Equity Scholarships to ensure inclusivity.</div>
+            <div class="action-buttons" layout center-justified center vertical>
+              <span class="price">$0-250</span>
+              <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdY9msPgujCvlVYP6jsOTO1t1WR8OEStIQA4HcsnkgxyRhQVw/viewform">
+                <paper-button class="cta-button animated icon-right"
+                  primary
+                  invert
+                  animated
+                  raised
+                  >
+                  <span>Apply Now</span>
+                  <iron-icon icon="hoverboard:arrow-right-circle"></iron-icon>
+                </paper-button>
+              </a>
+            </div>
+          </div>
+          <div class="number-sect">
+            <div class="number">3.</div>
+            <div class="number-text">Student Scholarships to make sure we have young voices and minds present.</div>
+            <div class="action-buttons" layout center-justified center vertical>
+              <span class="price">$250</span>
+              <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSetSldcR1CJYhmhRuyshB8hr8rjdRk0C-Spu0g15va5mGZWmA/viewform">
+                <paper-button class="cta-button animated icon-right"
+                  primary
+                  invert
+                  animated
+                  raised
+                  >
+                  <span>Apply Now</span>
+                  <iron-icon icon="hoverboard:arrow-right-circle"></iron-icon>
+                </paper-button>
+              </a>
+            </div>
+          </div>
+          <div class="number-sect">
+            <div class="number">4.</div>
+            <div class="number-text">Community Organizer
+                Scholarships for people
+                founding or involved with
+                local RxC chapters.</div>
+            <div class="action-buttons" layout center-justified center vertical>
+              <span class="price">$400</span>
+              <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSd1TR0wmH6F5EvJ1A7ZjBHh6MWzHgKR31qlxtrO4188yuy3eA/viewform">
+                <paper-button class="cta-button animated icon-right"
+                  primary
+                  invert
+                  animated
+                  raised
+                  >
+                  <span>Apply Now</span>
+                  <iron-icon icon="hoverboard:arrow-right-circle"></iron-icon>
+                </paper-button>
+              </a>
+            </div>
+          </div>
+        </section>
+      </div>
+        <p class="bottom-section container-narrow">Scholarships are based on specific eligibility criteria. See requirements in application descriptions. Scholarship tickets will grant you the same access as a general admission&nbsp;ticket.</p>
+    </div>
+  </template>
+
+  <script>
+    class ScholarshipBlock extends ReduxMixin(Polymer.Element) {

```

Comment:
Since this element isn't using any data from redux, you can extend it straight from `Polymer.Element`, and skip the mixin import.